components/helpers/interfaces.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 
-export type IFilter = 'All' | 'Open' | 'Voting' | 'Closed'
+export type IFilter = 'All' | 'open' | 'voting' | 'closed' | 'schedule'
```

Comment:
why is `All` uppercase?

Reply:
because that value is not being returned by the backend, it might as well be in uppercase.